aeproject-cli/aeproject-init/artifacts/exampleTests.js

```diff
@@ -0,0 +1,72 @@
+/*
+ * ISC License (ISC)
+ * Copyright (c) 2018 aeternity developers
+ *
+ *  Permission to use, copy, modify, and/or distribute this software for any
+ *  purpose with or without fee is hereby granted, provided that the above
+ *  copyright notice and this permission notice appear in all copies.
+ *
+ *  THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES WITH
+ *  REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY
+ *  AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT,
+ *  INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM
+ *  LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR
+ *  OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR
+ *  PERFORMANCE OF THIS SOFTWARE.
+ */
+const fs = require('fs')
+const { Universal: Ae, MemoryAccount, Node } = require('@aeternity/aepp-sdk');
+
+const EXAMPLE_CONTRACT = fs.readFileSync('./contracts/ExampleContract.aes', 'utf8')
+const NETWORKS = require('../config/network.json');
+const NETWORK_NAME = "local";
+
+describe('Example Contract', () => {
+    let contract;
+    let hamsterName;
+
+    before(async () => {
+        const account = MemoryAccount({ keypair: wallets[0] });
```

Comment:
where is wallets coming from here, I propose to make this more of an explicit import

Reply:
I would prefer to leave it as is right now and address this topic in the bigger refactoring. this is related to the funded wallets and they are provided in the tests through this line: https://github.com/marc0olo/aepp-aeproject-js/blob/6ede4d647b80131307f8b2096923677a44705b2d/aeproject-cli/aeproject-test/aeproject-test.js#L76

Reply:
what way would you prefer here? we need to consider:
- funding the wallets
- providing possibility to access the funded wallets in tests

ideally the dev could provide a mnemonic to use. but I want to avoid having the dev to define that twice for different cli commands. would this be a way to go in order to have that implemented quickly?
- use the mnemonic based on an ENV variable -> fallback to default mnemonic in case it's missing
    - would be used for funding and in the tests
- provide a new command that allows to fund additionall wallets quickly based on a mnemonic and additionally providing the amount of derivations

Reply:
I was just thinking about explicit import instead of magic aeproject stuff.

as done here: https://github.com/thepiwo/aeproject-boilerplate/blob/master/test/exampleTest.js#L3

Reply:
same for the chai assert probably.

then IDEs can use their native mocha integration for testing

Reply:
ok I mean if this "overwrites" the magic aeproject stuff I can add that quickly. everything else should be addressed in the bigger refactoring.

as we don't release a separate package anymore I will add that json file to the new config folder where I also placed the networks and refer to this in the exampleTest.

would this be ok?

Reply:
I guess I need to populate the package.json of the default template with chai and mocha, too. right?

Reply:
see recent commit. are we fine with that? :)

Reply:
looks good as far as I can see, haven't tried it out